index.html

```diff
@@ -77,7 +78,7 @@ <h2 class="intro connect">
                 </li>
                 <li>
                     <a href="">
-                        <img src="images/peace.svg" alt="social-media icon">
+                        <img src="images/peace.svg" alt="angellist icon">
                     </a>
```

Comment:
https://github.com/Dagic-zewdu/portifolio/blob/93d47c3e2fe5e5b4b5bebc01f6f60cd12e89ac17/index.html#L86-L88

![f](https://user-images.githubusercontent.com/57562869/151126316-249a19ed-5730-442c-b03f-61ac1af75c9e.png)


- [OPTIONAL ] The whole image `width and height` must be displayed correctly on the desktop version, Kindly fix that and make sure it display correctly before you merge this PR